src/services/wallets/JsonWalletConnector.ts

```diff
@@ -58,4 +66,35 @@ export class JsonWalletConnector implements ArweaveWalletConnector {
   async getWalletBalanceAR(): Promise<string> {
     throw Error('Not implemented!');
   }
+  async getWalletANTs(
+    approvedSourceCodeTransactions: string[],
+    cursor?: string,
+  ): Promise<{ ids: string[]; cursor?: string }> {
+    const address = await this.getWalletAddress();
+    const fetchedANTids: Set<string> = new Set();
+    let newCursor: string | undefined = undefined;
+
+    // get contracts deployed by user, filtering with src-codes to only get ANT contracts
+    const deployedResponse = await this._arweave.api.post(
+      '/graphql',
+      deployedAntQuery(address, approvedSourceCodeTransactions, cursor),
+    );
+    if (deployedResponse.data.data?.transactions?.edges?.length) {
+      deployedResponse.data.data.transactions.edges
+        .map((e: any) => ({
+          id: e.node.id,
+          cursor: e.cursor,
+        }))
+        .forEach((ant: { id: string; cursor: string }) => {
+          fetchedANTids.add(ant.id);
+          if (ant.cursor) {
+            newCursor = ant.cursor;
+          }
+        });
+    }
+    return {
+      ids: [...fetchedANTids],
+      cursor: newCursor,
+    };
+  }
```

Comment:
```suggestion
```

Reply:
will remove the majority but will have to stub this after updating the global wallet type i think. Unless we want to remove the json connector completely.

Reply:
you shouldn't have to stub it since `getWalletAnts` is no longer in the `ArweaveWalletConnector` interface

Reply:
removed JsonWalletConnector

Reply:
we don't need to delete the class, just remove it from the connect modal and anywhere else it's used.